taskcluster/fenix_taskgraph/target_tasks.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from taskgraph.target_tasks import _target_task, standard_filter
+
+# XXX We're overwritting the default target_task while all tasks are ported to taskgraph
```

Comment:
`XXX` :eyes: :stuck_out_tongue:

Reply:
Hahahaha, I had never connected the dots between this kind of code comment and the rest of the Internet 😅

(For any further reader: https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments)